images/custom-scorecard-tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the custom-scorecard-tests binary
+FROM --platform=x86_64 golang:1.17.9 as builder
+ARG TARGETARCH
+
+WORKDIR /workspace
+
+# Copy the go source
+COPY . .
+
+# Build
+RUN GOOS=linux GOARCH=$TARGETARCH BUILD_DIR=/workspace make GO_REQUIRED_MIN_VERSION:= build/custom-scorecard-tests
```

Comment:
The "FROM" image should be linux, so is GOOS needed?
Is GO_REQUIRED_MIN_VERSION:= needed?

Reply:
I don't think they are needed, will remove them